docker/AcceptanceTests/Dockerfile

```diff
@@ -1,3 +1,7 @@
+# How to execute:
+# $ docker build -t fiware-aiakos-acceptance .
+# $ docker run --name fiware-aiakos-acceptance fiware-aiakos-acceptance
```

Comment:
what about the environment variables needed? If we execute it by docker run -e option is required...